Zend/tests/errmsg_028.phpt

```diff
@@ -9,4 +9,4 @@ class self {
 echo "Done\n";
 ?>
 --EXPECTF--
-Fatal error: Cannot use 'self' as class name as it is reserved in %s on line %d
+Fatal error: Cannot use 'self' as a class name as it is reserved in %s on line %d
```

Comment:
Unrelated and can be done in a follow-up, but those should be double quotes instead of single quotes.
```suggestion
Fatal error: Cannot use "self" as a class name as it is reserved in %s on line %d
```